non_linear/graphs/floyd_warshall/floyd_warshall.py

```diff
@@ -0,0 +1,40 @@
+def adjacencyListToMatrix(adjList, vertices):
+    matrix = [[float('inf')] * vertices for _ in range(vertices)]
+    for i in range(vertices):
+        matrix[i][i] = 0  # Distance to self is 0
+    for u in adjList:
+        for v, weight in adjList[u]:
+            matrix[u][v] = weight
+    return matrix
+
+
+def floydWarshall(vertices, graph):
+    dist = [[graph[i][j] for j in range(vertices)] for i in range(vertices)]
```

Comment:
This can be more pythonic too

```suggestion
    dist = [row[:] for row in graph]
```